CMake/SMDefs.cmake

```diff
@@ -29,8 +29,13 @@ else()
     set(SM_VERSION_GIT
         "${SM_VERSION_MAJOR}.${SM_VERSION_MINOR}.${SM_VERSION_PATCH}")
   else()
-    set(SM_VERSION_GIT
-        "${SM_VERSION_MAJOR}.${SM_VERSION_MINOR}.${SM_VERSION_PATCH}-git-${SM_VERSION_GIT_HASH}")
+    if(WITH_NIGHTLY_RELEASE)
+      set(SM_VERSION_GIT
+        "${SM_VERSION_MAJOR}.${SM_VERSION_MINOR}.${SM_VERSION_PATCH}-git-${SM_VERSION_GIT_HASH}-NIGHTLY")
```

Comment:
Let's move the NIGHTLY before the `-git-`, and then also add BETA for the below condition in a similar fashion.

Reply:
https://github.com/itgmania/itgmania/pull/782/commits/0d6a2f71959e8064b76aad46f8d965f5670d19f2